src/spetlr/filehandle/file_handle.py

```diff
@@ -76,20 +91,26 @@ def read(self) -> DataFrame:
         return reader.load(self._location)
 
     def read_stream(self) -> DataFrame:
-        reader = (
-            Spark.get()
-            .readStream.format("cloudFiles")
-            .option("cloudFiles.format", self._data_format)
-            .option("cloudFiles.schemaLocation", self._schema_location)
-        )
+        reader = Spark.get().readStream.format("cloudFiles")
 
         if self._options is not None:
```

Comment:
Here we can just say `if self._options` also refactor this for the `read` method. Just to stay consistent in the module.